website/content/plugins/drivers/podman.mdx

```diff
@@ -400,6 +434,40 @@ The `podman` driver implements the following [capabilities](/nomad/docs/concepts
   }
   ```
 
+- `security_opt` - (Optional) A list of security-related options that are set
+  in the container.
```

Comment:
```suggestion
- `security_opt` - (Optional) A list of security-related options that Podman sets
  in the container.
```

Style: use active voice - and I guessed that Podman sets these options in the container after reading some of the Podman docs.